timeseries_example.ipynb

```diff
@@ -79,7 +79,7 @@
    "metadata": {},
    "source": [
     "## Step 1: Select\n",
-    "In order to work with the timeseries tools, **be sure to select both \"Historical Climate\" and one of the SSP timeseries options.** The historical data will be automatically appended to a SSP time series when both are selected. **You'll also need to select \"Area Average\".**\n",
+    "In order to work with the timeseries tools, **be sure to select both \"Historical Climate\" and one of the SSP timeseries options.** The historical data will be automatically appended to a SSP time series when both are selected. **You'll also need to compute an area average over the grid cells. This option is found in the second tab, where you'll select **yes** under the area average selection.\n",
```

Comment:
Just missing an extra set of ** here because the "yes" ends up being unbolded with additional ** afterwards. Maybe put the missing pair after grid cells in the first sentence